var/spack/repos/builtin/packages/xfsprogs/package.py

```diff
@@ -29,6 +29,7 @@ class Xfsprogs(AutotoolsPackage):
     depends_on("gettext@:0.21.1", when="@:6.3")
     depends_on("uuid")
     depends_on("util-linux")
+    depends_on("liburcu", when="@6.11.0:")
```

Comment:
Was this introduced with 6.11 or with 6?

Reply:
Looks like 6.0.0 needs it too. I modified.